### DISCUSSION TITLE
Will this counter be deprecated or only harvest template will be deprecated?

### DISCUSSION THREAD

**Discussion by jmg011** [NetApp/harvest#1686] [General]
Hi,

**From these release notes:** https://github.com/NetApp/harvest/discussions/1020

IMPORTANT The conf/zapiperf/cdot/9.8.0/object_store_client_op.yaml ZapiPerf template is being deprecated in this release and will be removed in the next release of Harvest. No dashboards use the counters defined in this template and all counters are being deprecated by ONTAP. If you are using these counters, please create your own copy of the template.

**Question:** 

Is the object_store_client_op object is going to be deprecated from ZAPI and is that the reason why this template is deprecated/removed in latest versions of Harvest?

**Comment by cgrinds** [comment]
hi @jmg011 it was deprecated because no dashboards were consuming these counters and while the Harvest team was creating REST templates that match the ZAPI ones, we discovered that ONTAP deprecated these counters when they moved from ZAPI to REST so there's no way for Harvest to collect them via REST. 

Per our conversation last week, we'll leave the template in Harvest and not remove this template. Keep in mind these counters won't exist on the REST side.

**Reply by jmg011** [reply]
@cgrinds - Understood. Thank you. If the counters are going to be deprecated from ONTAP, are we going to have the new counters in ONTAP? Any idea on that front?

**Reply by cgrinds** [reply]
@jmg011 ONTAP moved some of the counters, which we have already included in Harvest templates. They also removed some of the counters without replacements. There isn't anything Harvest can do about those. This [knowledge base article](https://kb.netapp.com/Advice_and_Troubleshooting/Data_Storage_Software/ONTAP_OS/FAQs_on_ZAPI_to_ONTAP_REST_API_transformation_for_CPC_(Customer_Product_Communiques)_notification) has details on how to address any gaps.

**Reply by jmg011** [reply]
@cgrinds  - Thank you. This KB is useful. I will follow up with ONTAP team.
